### ISSUE TITLE
BUG: curtime() vs now()

### ISSUE THREAD

**Issue by joelbrock** [CORE-POS/IS4C#505]
$db->curtime() throws an error.  THe correct answer (for MySQL anyway) is $db->now():

UPC.php:348
```php
        if ($row["idEnforced"] > 0){

            $restrictQ = "SELECT upc,dept_ID FROM dateRestrict WHERE
                ( upc='{$row['upc']}' AND
                  ( ".$db->datediff($db->now(),'restrict_date')."=0 OR
                    ".$db->dayofweek($db->now())."=restrict_dow
                  ) AND
                  ( (restrict_start IS NULL AND restrict_end IS NULL) OR
                    ".$db->now()." BETWEEN restrict_start AND restrict_end
                  )
                 ) OR
                ( dept_ID='{$row['department']}' AND
                  ( ".$db->datediff($db->now(),'restrict_date')."=0 OR
                    ".$db->dayofweek($db->now())."=restrict_dow
                  ) AND
                  ( (restrict_start IS NULL AND restrict_end IS NULL) OR
                    ".$db->now()." BETWEEN restrict_start AND restrict_end
                  )
                )";
```

**Comment by joelbrock** [comment]
https://github.com/joelbrock/HARVEST_CORE/commit/73ceb2d0e5b76b272ffbea4cec48b72b5c6178e8

**Comment by gohanman** [comment]
I like this better as a fix (also patched back into 1.3):
4d48e98891945e00d1c98a963d25d9b5c6551940

I worry that "WHERE [DATETIME] BETWEEN [TIME] AND [TIME]" is the kind of thing that might not be particularly reliable.
